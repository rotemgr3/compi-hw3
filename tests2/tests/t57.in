 
//piazza 119 - should be ok  +
void foo (int a) {int booga = 5;}
void test (int a) {int booga = 5;}
void main() {int booga = 5;}