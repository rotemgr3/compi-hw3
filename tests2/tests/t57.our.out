---end scope---
a INT -1
booga INT 0
---end scope---
a INT -1
booga INT 0
---end scope---
booga INT 0
---end scope---
print (STRING)->VOID 0
printi (INT)->VOID 0
foo (INT)->VOID 0
test (INT)->VOID 0
main ()->VOID 0
